tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.md

```diff
@@ -181,7 +181,20 @@ This test requires that a vSphere server is running and available
 8. Docker container created with network attached with net1, got ip address inside of network range
 9. Docker create another container, and link to previous one, can talk to the the first container successfully
 
-
+## Container Firewalls
+1. Create an open container and verify another open container can connect to it on arbitrary ports.
+2. Create a closed container and verify an open container cannot connect to it on an arbitrary port.
```

Comment:
Adding:
* Create a Closed container on an external network, and on bridge network - ensure that a container on the same bridge network _can_ connect to the Closed container.